src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.List;
+
+public interface CallbackProcessor {
```

Comment:
why do you need this interface?

Reply:
Well not really ... its one of those springy things about object contracts.
Could remove it if ya want

Reply:
would prefer, seems unneeded to me

Reply:
It seems I got then to  branches mixed in this PR ... just give me a sec